AFNetworking/PrivacyInfo.xcprivacy

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Does this file need to be in a particular place to be correctly consumed/reviewed by Apple like under Resources or Support Files or Settings.bundle? Assuming this is ok, just want to set a precedent for consistency across internal repos.

Reply:
ya true i'll put it in a folder called Resources for consistency across other repos. not sure if it affects Apple's review but better safe than sorry!

Reply:
oh wait i had put it in Supporting Files not sure why that's not being reflected here ...

Reply:
oh right i think xcode automatically groups these in a folder called Supporting Files on xcode but its not a real folder in Finder so it's confusing. ill just move it to a new Resources folder